src/context.ts

```diff
@@ -216,25 +216,31 @@ export class Context<E extends WebhookPayloadWithRepository = any>
    * Config files can also specify a base that they extend. `deepMergeOptions` can be used
    * to configure how the target config, extended base, and default configs are merged.
    *
+   * Additionally, if there is no config file, but there is a repo in the org named .github,
+   * it will be used as a base repository and file from there will be loaded. This can be
+   * opted-out by setting `loadBaseWhenMissing` to false
```

Comment:
I think if nothing else, would you agree that at least adding this to the documentation would be good?

Reply:
Please give me a few days, I'll get back to you on that